sdk/monitor/monitor-opentelemetry-exporter/src/utils/breezeUtils.ts

```diff
@@ -33,7 +33,9 @@ export function isRetriable(statusCode: number): boolean {
     statusCode === 408 || // Timeout
     statusCode === 429 || // Too many requests
     statusCode === 500 || // Server Error
-    statusCode === 503 // Server Unavailable
+    statusCode === 502 || // Bad Gateway
```

Comment:
Please add this in the changelog
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-opentelemetry-exporter/CHANGELOG.md